src/components/Input/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { Fragment, Ref } from 'react';
+
+import styles from './styles.module.scss';
+
+interface InputProps {
+  name: string;
+  label: string;
+  error?: string;
+  inputRef?: Ref<HTMLInputElement>;
+  type?: string;
+}
+
+function Input({ error, label, name, inputRef, type }: InputProps) {
+  return (
+    <Fragment>
+      <label htmlFor={name} className={styles.label}>
+        {label}
+      </label>
+      <input className={styles.input} name={name} type={type} ref={inputRef} />
```

Comment:
when you reach tests section, you will have problem testing input because label is associated to the input via `id` property which is missing, the previous `htmlFor` attached that label to the input component with the id, not the name